packages/Panel/src/components/Dialog/Dialog.js

```diff
@@ -4,7 +4,7 @@ import * as types from "../../types";
 import * as sc from "./Dialog.styles";
 
 function Dialog(props) {
-  const [isAnimating, setIsAnimating] = React.useState(false);
+  const [isAnimating, setIsAnimating] = React.useState(true);
```

Comment:
When this is `false` there is a render of the panel being open on first render pass. This can be observed on[ this story ](http://storybooks.highbond-s3.com/paprika/master/?path=/story/messaging-panel-examples--push-content-panel)